src/components/card/card.style.ts

```diff
@@ -29,20 +42,26 @@ const StyledCard = styled.div<StyledCardProps>`
     outline: none;
     ${margin}
 
-    ${interactive &&
-    css`
+    ${
+      interactive &&
+      css`
       cursor: pointer;
 
       :hover,
       :focus {
-        box-shadow: var(--boxShadow100);
+        box-shadow: ${
+          hoverBoxShadow ? `var(--${hoverBoxShadow})` : `var(--boxShadow100)`
```

Comment:
As above, just set a default `hoverBoxShadow` function argument value to `var(--boxShadow100)`

Reply:
See the above for resolution comment